### PR TITLE
Add copy/pastable release text

### PR DESCRIPTION
**Pull request by kytrinyx**

### CODE REVIEW

**Comment by Tonkpils** [comment]
Do we need to update this from master since the other one was merged? Now sure if this overrides the other commit.

**Comment by kytrinyx** [comment]
No, there shouldn't be any conflicts.

File: RELEASE.md [hunk 1]
```diff
@@ -49,28 +51,25 @@ VERSION matches the value of the `Version` constant.
 
 Upload all the binaries from `release/*`.
 
-## Update Homebrew
+Past the release text and describe the new changes:
```
Comment (Tonkpils) [inline RELEASE.md]: I think you mean `Paste` here.

**Comment by Tonkpils** [comment]
Cool, I'll merge this and fix the typo :+1:

**Comment by kytrinyx** [comment]
Thank you :)

**Pull request merged**